diff --git a/dist/generator.d.ts b/dist/generator.d.ts
index 5b34c70c37e1f58d71162ed7360ea916b4f6e73f..b6d5c4978be8ded2498bf6a2e154b68f4691786e 100644
--- a/dist/generator.d.ts
+++ b/dist/generator.d.ts
@@ -1,4 +1,4 @@
-import {Config} from "tailwindcss/types/config";
+import { Config } from "tailwindcss";
 
 export type GenerateTypes = {
   (theme: Config["theme"]): void;
diff --git a/dist/transformer.d.ts b/dist/transformer.d.ts
index 508be4d8176486bfbbccd0b8cf297465c8efd15a..67b205cc3607870ae9500c300d46be31709d48b7 100644
--- a/dist/transformer.d.ts
+++ b/dist/transformer.d.ts
@@ -1,7 +1,7 @@
-import type {Config} from "tailwindcss/types/config";
-import type {DefaultTheme} from "tailwindcss/types/generated/default-theme";
+import type { Config } from "tailwindcss";
+import type DefaultTheme from "tailwindcss/defaultTheme";
 
-export type DefaultScreens = keyof DefaultTheme["screens"];
+export type DefaultScreens = keyof (typeof DefaultTheme)["screens"];
 
 export type WithTV = {
   <C extends Config>(tvConfig: C, config?: TVTransformerConfig): C;
